gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -347,6 +347,32 @@ public static boolean intersectSegmentCircle (Vector2 start, Vector2 end, Circle
 		return v2a.len2() <= circle.radius * circle.radius;
 	}
 
+	/** Returns whether the given {@link Frustum} intersects a {@link BoundingBox}.
+	 * @param frustum The frustum
+	 * @param bounds The bounding box
+	 * @return Whether the frustum intersects the bounding box*/
+	public static boolean intersectFrustumBoundsFast(Frustum frustum, BoundingBox bounds) {
```

Comment:
Does it need `Fast` in the name?

Reply:
I usually use the `Fast` suffix when it returns a boolean. I can remove it.

Reply:
Not sure how fast in the name is related to returning a boolean? :) Seeing "fast" in the name makes me think the function is taking shortcuts and maybe losing precision or accuracy compared to some other way of doing the same operation.

Reply:
typically "non fast" methods returns more information and need more calculations (eg. collision point on a sphere), "fast" method only detect collision and simply returns a boolean.
I can't see what kind of information frustum/AABB collision detection could return so i agree, "fast" could be removed.